src/index.js

```diff
@@ -0,0 +1,36 @@
+import './style.css';
+import { renderContainer, toAddScore } from './modules/data.js';
+
+let gameIdentity = '';
+fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games', {
+
+  method: 'POST',
+  headers: {
+    'Content-type': 'application/json; charset=UTF-8',
+  },
+  body: JSON.stringify({
+    name: 'Handball tx',
+  }),
+}).then((res) => res.json())
+  .then((gameID) => {
+    const response = gameID.result;
+    gameIdentity = response.substring(14, response.lastIndexOf(' '));
+  });
```

Comment:
- [ ] You have implemented your api in a very great way 😊 but there is a problem. When we add something in your api through your form and refresh it and then api doesn't return us the old data which we added before. The problem is everytime your run your application it creates a new game id. A new game id means a new api. Kindly use only one game id even after refreshing the page 👍

Reply:
Fixed.